src/scss-styles/responsive.scss

```diff
@@ -0,0 +1,5 @@
+@media (max-width:767px){
+    .text-mb-center{
```

Comment:
For this we need media query? I'm not sure. I think we can replace it with "text-center" "text-md-center" like bootstrap classes

Reply:
unwanted media quires removed , i have updated bootstrap mam